packages/odyssey-react/src/components/Field/Field.module.scss

```diff
@@ -11,8 +11,6 @@
  */
 
 .root {
-  display: flex;
-  flex-direction: column;
   max-width: $max-line-length;
   margin-block-end: $spacing-m;
```

Comment:
Box should have an API for the margin here:
```
marginBottom="m"
```
It is tricky because we're using physical properties in the API here but all our styles are logical 🤯

Reply:
I thought we'd decided to eschew using utilities for our components right now because we love the per-component value mapping.

Reply:
@edburyenegren-okta you're right thanks for the reminder!